src/test/skript/tests/syntaxes/expressions/ExprExplosiveYield.sk

```diff
@@ -11,3 +11,16 @@ test "creeper explosive yield":
 	set explosive yield of last spawned creeper to 10
 	assert explosive yield of last spawned creeper is 10 with "a creeper's explosive yield should be 10 if it is set to 10"
 	delete last spawned creeper
+
+test "ghast explosive power":
```

Comment:
needs tests for NaN and infinities, and just normal max value limits like a long that's bigger than the int max or negative numbers